### ISSUE TITLE
Zabbix Proxy using Sqlite

### ISSUE THREAD

**Issue by drequena** [dj-wasabi/puppet-zabbix#184]
I couldn't find a way to install Zabbix::Proxy using Sqlite as DB.
I think this is a very common setup, please, add this support. ;)

**Comment by dj-wasabi** [comment]
Hi,

You can do it like this:
```
class { 'zabbix::proxy':
  zabbix_server_host => 'zabbix.example.com',
  database_type      => 'sqlite',
  database_name      => '/tmp/database',
}
```

I'll update the documentation about this.

**Comment by drequena** [comment]
Hi,
I did try do this, but maybe I did something wrong.
I just looked at my reports and I found this:

Could not retrieve catalog from remote server: Error 400 on SERVER: Unrecognized database type for proxy: sqlite at /etc/puppet/environments/production/modules/zabbix/manifests/proxy.pp:461 on node lx-vm002-eqnx.lef.intra

I'll give another try without Foreman, maybe I screwed up on overwrite parameters.
Thanks.
